Makefile

```diff
@@ -39,13 +39,18 @@ apidiff::
 
 test-schema::
 	@set -a; \
-	. .env; \
-    if [ -z "$$writeDbUrl" ]; then \
-		echo "writeDbUrl is not set in .env - using test db and running migrations"; \
-		writeDbUrl=postgresql://postgres:example@localhost:21300/postgres; \
-		make migrate; \
-	fi; \
+	writeDbUrl=postgresql://postgres:example@localhost:21300/postgres; \
```

Comment:
after these changes does this still hold?

https://github.com/AudiusProject/api?tab=readme-ov-file#to-update-schema-after-migration-changes-and-run-tests

Reply:
Good catch - I'll update those. Just need `make test-schema && make test` now